### ISSUE TITLE
BUG: MySQL CURRENT_TIMESTAMP() default fails if field was touched

### ISSUE THREAD

**Issue by henkedk** [beekeeper-studio/beekeeper-studio#869]
**Describe the bug**
Adding a new row, accidentally editing a field that is supposed to hold a date, deleting all entered values, leaving a field with a default value of a MySQL fucntion (eg. CURRENT_TIMESTAMP()) empty will result in the insert failing.

**To Reproduce**
1. Go to 'VIEW DATA' on a table
2. Click on '+' at the bottom to add a row
3. Add data in the fields, enter something in the date field, but delete again before apply so field is empty
4. Click APPLY
5. Insert fails

**Expected behavior**
Insert succeeding, as field is supposed to use DEFAULT VALUE defined in structure.

**Version Information (please complete the following information):**
 - OS: Zorin OS 16 (Ubuntu 20.04)
 - App Version 2.1.5

**Additional context**
Adding the function manually in the field (eg CURRENT_TIMESTAMP()) will also fail, this time it will be "INCORRECT DATE TIME VALUE"

**Comment by rathboma** [comment]
Yes, this is a limitation of our editing interface. The only alternative would be to require quotes `'` around all strings, which would also be bad.

I think in future we should add a right click -> current timestamp or add an exception for CURRENT_TIMESTAMP in mysql, or maybe adding a right click -> edit as raw SQL.

I'll think about a way to do this.

This is on purpose -- this behavior also prevents accidental sql injection :-D.

**Comment by GingerAdonis** [comment]
Is there any update on this? It's baffling to me that I can't simply set `CURRENT_TIMESTAMP` or use MySQL functions.

Thank you. :)
